lib/widgets/dialogs/delete_place_dialog.dart

```
@@ -24,9 +26,17 @@ class DeletePlaceDialog extends ConsumerWidget {
 
       // Unsubscribe from server
       debugPrint("unregister from server for place ${myPlace.name}");
-      await alertApi.unregisterArea(
-        subscriptionId: myPlace.subscriptionId,
-      );
+      try {
+        await alertApi.unregisterArea(
+          subscriptionId: myPlace.subscriptionId,
+        );
+      } on UnregisterAreaError {
+        ErrorLogger.writeErrorLog(
```

Please don't do this kind of silent error handling. Instead inform the user something went wrong so they can act upon it, for example by suggesting they should try again later.

---

I see your point, but I'm not sure if we should inform the user here. The user just wants to delete his subscription and should be able to do this even if the device is offline. We should add the deletion request to the waiting queue (#135) and just try it again later. But I will think about this again.

---

I changed my mind. A waiting queue is not necessary. It is way easier if users can only unsubscribe with an internet connection. We now show a snack bar to inform users to check their internet connection.

---

Yeah that seems way easier and good enough. In general actions that require networking can just inform the user there is no network.
We could even have some `ConnectedButton` that's only enabled when an active connection to the backend is detected. Pressing it while it's disabled then informs the user of the networking issue. We do this at work, works quite nicely.